dist/combobox/ds6/combobox.css

```diff
@@ -0,0 +1,65 @@
+.combobox {
+  display: inline-block;
+  box-sizing: border-box;
+  width: 100%;
+}
+.combobox__control > input {
+  background-color: #ffffff;
+  border: 1px solid #c7c7c7;
+  box-sizing: border-box;
+  color: transparent;
+  cursor: default;
+  font-family: inherit;
+  font-size: 16px;
+  height: 40px;
+  padding: 0 16px;
+  text-shadow: 0 0 0 #111820;
+  vertical-align: middle;
+  width: 100%;
+}
+.combobox__control > input:focus {
+  border-color: #4295ff;
+}
+.combobox__control > input[disabled],
+.combobox__control > input[aria-disabled="true"] {
+  border-color: #c7c7c7;
+  color: transparent;
+  text-shadow: 0 0 0 #c7c7c7;
+}
+.combobox__control > input[aria-expanded="true"] {
+  border-bottom: 0;
+  padding-bottom: 1px;
+}
+.combobox__control > input::-moz-selection {
+  background: #ffffff;
+}
+.combobox__control > input::selection {
+  background: #ffffff;
+}
+.combobox__control + div.combobox__options {
+  display: none;
+  position: absolute;
+  z-index: 1;
+}
+.combobox__control--expanded + div.combobox__options {
+  display: block;
+}
+span.combobox__arrow {
+  background-image: url('data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20width%3D%278%27%20height%3D%275%27%3E%3Cpath%20fill%3D%22%2311820%22%20d%3D%22M0.758714126%2C0.14644663%20C0.585147774%2C-0.048815516%200.303741117%2C-0.0488155166%200.130174765%2C0.146446629%20C-0.0433915877%2C0.341708774%20-0.0433915882%2C0.658291264%200.130174763%2C0.85355341%20L3.68573029%2C4.85355339%20C3.85929664%2C5.04881554%204.1407033%2C5.04881554%204.31426965%2C4.85355339%20L7.86982523%2C0.853553392%20C8.04339159%2C0.658291247%208.04339159%2C0.341708758%207.86982524%2C0.146446611%20C7.69625889%2C-0.0488155358%207.41485223%2C-0.0488155372%207.24128588%2C0.146446608%20L3.99999997%2C3.79289322%20L0.758714126%2C0.14644663%20Z%22%3E%3C/path%3E%3C/svg%3E');
+  background-repeat: no-repeat;
+  background-position: right center;
+  content: "";
+  height: 40px;
+  margin-left: 8px;
+  position: absolute;
+  right: 16px;
+  top: 0;
+  width: 8px;
+}
+.combobox--expanded span.combobox__arrow {
+  -webkit-transform: rotate(180deg);
+          transform: rotate(180deg);
+}
+.combobox__borderless .combobox__control > input {
```

Comment:
Maybe we could move this modifier closer to the BEM element? i.e. `.combobox__control--borderless`? These days I'm trying to get the modifier as close to the element it effects as I can.

Reply:
Yes, I can move it. Totally valid.